backend/tests/conftest.py

```diff
@@ -0,0 +1,98 @@
+from collections.abc import Generator
+from typing import Any
+
+import paramiko
+import pytest
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives.asymmetric.rsa import RSAPrivateKey, RSAPublicKey
+from faker import Faker
+from faker.providers import DynamicProvider
+from sqlalchemy.orm import Session as OrmSession
+
+from mirrors_qa_backend.db import Session, models
+from mirrors_qa_backend.enums import StatusEnum
+
+
+@pytest.fixture
+def dbsession() -> Generator[OrmSession, None, None]:
+    with Session.begin() as session:
+        # Ensure we are starting with an empty database
+        engine = session.get_bind()
+        models.Base.metadata.drop_all(bind=engine)
+        models.Base.metadata.create_all(bind=engine)
+        yield session
+        session.rollback()
+
+
+@pytest.fixture
+def fake_data(faker: Faker) -> Faker:
```

Comment:
this name does not inform about what it returns

Reply:
It's a fixture that returns fake data. It relies on `Faker` but adds new providers to it so we can generate valid status(es), country names and other data that Faker doesn't know but is specific to our application

Reply:
The parameter `faker` is the  name of the fixture that `Faker` (the library) provides to pytest. I couldn't rename it since I wanted to use it but only to give it new data. What do you suggest I should call the modified `faker`? Sorry, I am terrible at naming things 😅

Reply:
Okay. Thanks